test/ct_test.c

```diff
@@ -18,8 +18,7 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 #include "testutil.h"
-
-#ifndef OPENSSL_NO_CT
```

Comment:
Uh, you're right - this is needed; I've removed the `#else` though because none of our other tests define a trivial `main` I believe (rather, tests are skipped by the harness).

However `no-ct` is also broken in master...